module/definition.nix

```diff
@@ -132,15 +124,15 @@ let
 
 in
 {
-  options.detsys = {
+  options.detsys.vaultAgent = {
     defaultAgentConfig = mkOption {
-      description = "Default Vault agent configuration. Delegates to individual <code>agentConfig</code>s, if set.";
+      description = "Default Vault agent configuration. Defers to individual <code>agentConfig</code>s, if set.";
```

Comment:
Best part of the whole https://github.com/DeterminateSystems/nixos-vault-service/commit/98a95b167fcec7333849355163d129a44aee2736 diff right here.